component/loki/relabel/relabel_test.go

```diff
@@ -0,0 +1,100 @@
+package relabel
```

Comment:
Can you write a benchmark for this component, please?

Reply:
That's fine, I just want us to start having benchmarks to make it easier to track performance over time 👍 We don't need to worry about if 5ms per series is too high or not for now.